cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -6,6 +6,11 @@
 # If the operator needs to manage multiple clusters (in different namespaces), see the section below
 # for "cluster-specific resources". The resources below that section will need to be created for each namespace
 # where the operator needs to manage the cluster. The resources above that section do not be created again.
+#
+# Most of the sections are prefixed with a 'OLM' keyword which is used to build our CSV for an OLM (Operator Life Cycle manager)
```

Comment:
I think this line is sufficient. Just to keep it simple, no need for the other three lines after this.